CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+Version 1.4.0
+------
+* Updates to test, dependencies and internal tooling
```

Comment:
Should we start a habit of linking to the PRs?

Reply:
(also s/test/tests ?)

Reply:
It probably would be useful; we just need to be cognizant that some of the stuff contained within App CLI is not quite fully public yet (e.g., Argo-related).

Reply:
I'm tweaking the PR template to add changelog updates as a checkbox so we keep track of the relevant information as it comes in, so we can remind contributors to consider whether it's a change worth mentioning.